### PR TITLE
Added RuneLite account logout confirmation dialog

### PR DESCRIPTION
**Pull request by Infinitay**
Added a confirmation window when clicking to sign out of **RuneLite**.

### CODE REVIEW

File: runelite-client/src/main/java/net/runelite/client/account/SessionManager.java [hunk 1]
```diff
@@ -232,9 +234,17 @@ public void onLogin(LoginResponse loginResponse)
...
 	}
 
+	@Inject
+	RuneLite runeLite;
+
 	public void logout()
 	{
-		closeSession();
-		deleteSession();
+		if (JOptionPane.YES_OPTION == JOptionPane.showConfirmDialog(runeLite.getGui(),
```
Comment (deathbeam) [inline runelite-client/src/main/java/net/runelite/client/account/SessionManager.java]: Don't do this. It is perfectly fine to set "null" as parent for dialog, no need to inject RuneLite and call getGui on it.

File: runelite-client/src/main/java/net/runelite/client/account/SessionManager.java [hunk 2]
```diff
@@ -47,6 +38,17 @@
 import net.runelite.http.api.account.OAuthResponse;
 import net.runelite.http.api.ws.messages.LoginResponse;
 
+import javax.inject.Inject;
```
Comment (deathbeam) [inline runelite-client/src/main/java/net/runelite/client/account/SessionManager.java]: Fix your import settings: https://github.com/runelite/runelite/wiki/Code-Conventions

File: runelite-client/src/main/java/net/runelite/client/RuneLite.java [hunk 3]
```diff
@@ -100,6 +102,7 @@
 	private ClientSessionManager clientSessionManager;
 
 	Client client;
+	@Getter
```
Comment (deathbeam) [inline runelite-client/src/main/java/net/runelite/client/RuneLite.java]: Don't do this, we have Guice

File: runelite-client/src/main/java/net/runelite/client/RuneLite.java [hunk 4]
```diff
@@ -53,6 +49,12 @@
 import org.slf4j.LoggerFactory;
 import org.slf4j.MDC;
 
+import javax.inject.Singleton;
```
Comment (deathbeam) [inline runelite-client/src/main/java/net/runelite/client/RuneLite.java]: Fix your import settings: https://github.com/runelite/runelite/wiki/Code-Conventions

File: runelite-client/src/main/java/net/runelite/client/account/SessionManager.java (hunk 1 above)
Comment (Infinitay) [reply runelite-client/src/main/java/net/runelite/client/account/SessionManager.java]: It was so the new dialog would be centered and relative to RuneLite's location

**Comment by Infinitay** [comment]
>Removed changes from within SessionManager and RuneLite
Added the confirmation dialog to within the AccountPlugin (thanks deathbeam)

Once again, I passed `ui` into JOptionPane in oder for the pane to appear **centered and relative to the RuneLite client**.

**Pull request merged**